doc/arkode/guide/source/Usage/MRIStep/Custom_Inner_Stepper/Description.rst

```diff
@@ -79,6 +79,32 @@ Creating and Destroying an Object
       for details on how to attach member data and method function pointers.
 
 
+.. c:function:: int MRIStepInnerStepper_CreateFromSUNStepper(SUNStepper sunstepper, MRIStepInnerStepper* stepper)
+
+   This utility function wraps a :c:type:`SUNStepper` as an
+   :c:type:`MRIStepInnerStepper`.
+
+   :param sunctx: the SUNDIALS simulation context.
+   :param sunstepper: the c:type:`SUNStepper` to wrap.
+   :param stepper: a pointer to an MRI inner stepper object.
+
+   :retval ARK_SUCCESS: if successful
+   :retval ARK_MEM_FAIL: if a memory allocation error occurs
+
+   **Example usage:**
+
+   .. code-block:: C
+
+      SUNStepper sunstepper = NULL;
+      SUNStepper_Create(ctx, &sunstepper);
+      /* Attach content and functions to the SUNStepper... */
+
+      MRIStepInnerStepper inner_stepper = NULL;
+      flag = MRIStepInnerStepper_Create(sunstepper, &inner_stepper);
```

Comment:
How critical is it that the user fully configure the SUNStepper *before* they wrap it as an MRIStepInnerStepper?  If this copies existing function pointers then the order is important, in which case this documentation should contain a "note" to clarify the assumed order.  If the routine just attaches wrapper routines, and the SUNStepper could be modified after creating the MRIStepInnerStepper, then the current documentation should be sufficient.

Reply:
`MRIStepInnerStepper_CreateFromSUNStepper` wraps the `SUNStepper` and makes no copies of function pointers. The order in which `MRIStepInnerStepper_CreateFromSUNStepper` and the `SUNStepper_Set*` functions are called shouldn't matter.